FiveCalls/FiveCalls/AboutHtmlViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  AboutHtmlViewController.swift
+//  FiveCalls
+//
+//  Created by Alex on 2/6/17.
+//  Copyright © 2017 5calls. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AboutHtmlViewController : UIViewController, UIWebViewDelegate {
+    
+    @IBOutlet weak var contentView: UIWebView!
+    var contentName: String { get { return "whycall" } }
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+    
+    override func viewDidLoad() {
+        self.navigationController?.navigationBar.tintColor = .white
+        if let asset = NSDataAsset(name: contentName) {
+            let htmlString = String(data: asset.data, encoding: .utf8)!
+            contentView.loadHTMLString(htmlString, baseURL: nil)
+            contentView.delegate = self
+        }
+    }
+    
+    func webView(_ webView: UIWebView, shouldStartLoadWith request: URLRequest, navigationType: UIWebViewNavigationType) -> Bool {
+        switch navigationType {
+        case .linkClicked:
+            // Open links in Safari
+            guard let url = request.url else { return true }
+            
+            if #available(iOS 10.0, *) {
+                UIApplication.shared.open(url, options: [:], completionHandler: nil)
```

Comment:
Those second two parameters have defaults so you can do just `UIApplication.shared.open(url)`